src/_help/specification-support/doc-beta.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Beta
+---
+
+- TOC
+{:toc}
+
+Use the `x-beta` property inside an operation, a schema or a parameter object to identify it as beta.
+The `x-beta` property is a boolean.
+
+Adding or removing this property is a structural change.
+A change in a beta component is never identified as a breaking change.
+
+## Example usage
+
+Here under is an example of a beta operation, a beta request body and a beta schema attribute.
+
+```yaml
+  "paths": {
```

Comment:
Could examples use YAML so its easier to read? Also this says its yaml but is json so needs changing anyhow. [Boop](https://apps.apple.com/us/app/boop/id1518425043?mt=12) is a handy tool for switching JSON to YAML (and a million other things).

Reply:
I've change that, thanks!

> [Boop](https://apps.apple.com/us/app/boop/id1518425043?mt=12) is a handy tool for switching JSON to YAML

I use Dhall language tools for that which is very handy :smile: 
```
cat tmp/paths.json | json-to-dhall | dhall-to-yaml
```
You can do the opposite too:
```
cat tmp/paths.yaml | yaml-to-dhall | dhall-to-json
```